src/app/fddev/dev.c

```diff
@@ -119,6 +119,12 @@ update_config_for_dev( config_t * const config ) {
                                    NULL,
                                    "%s/vote-account.json",
                                    config->scratch_directory ) );
+
+  fd_topo_tile_t * gui = &config->topo.tiles[ fd_topo_find_tile( &config->topo, "gui", 0UL ) ];
```

Comment:
Does this need a check to make sure it found it? I think it's possible to run `fddev` with the GUI disabled in the TOML, right?